mifosng-android/src/main/java/com/mifos/api/local/MifosSQLCipherHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.mifos.api.local;
+
+import com.raizlabs.android.dbflow.config.DatabaseDefinition;
+import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener;
+import com.raizlabs.dbflow.android.sqlcipher.SQLCipherOpenHelper;
+
+public class MifosSQLCipherHelper extends SQLCipherOpenHelper {
+
+    public MifosSQLCipherHelper(DatabaseDefinition databaseDefinition,
+                                DatabaseHelperListener listener) {
+        super(databaseDefinition, listener);
+    }
+
+    @Override
+    protected String getCipherSecret() {
+        return "dbflow-rules";
```

Comment:
If we are to add it in a separate PR, do all the Cipher work together because if it gets squashed in this one it will be difficult to track in future. so remove the cipher related code from this pr.